src/components/Courses/tests/courses.test.js

```diff
@@ -0,0 +1,85 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { Switch } from 'react-router-dom';
+import { BrowserRouter as Router } from 'react-router-dom/cjs/react-router-dom.min';
+import { mockedAuthorsList, mockedCoursesList } from '../../../constants';
+import CourseForm from '../../CourseForm/CourseForm';
+import Courses from '../Courses';
+
+const mockedState_C_A = {
+	user: {
+		isAuth: true,
+		name: 'Test Name',
+		token: 'test',
+		role: 'admin',
+	},
+	courses: mockedCoursesList,
+	authors: mockedAuthorsList,
+};
+
+const mockedStore_C_A = {
+	getState: () => mockedState_C_A,
+	subscribe: jest.fn(),
+	dispatch: jest.fn(),
+};
+
+const mockedState_NoC_A = {
+	user: {
+		isAuth: true,
+		name: 'Test Name',
+		token: 'test',
+		role: 'admin',
+	},
+	courses: [],
+	authors: mockedAuthorsList,
+};
+
+const mockedStore_NoC_A = {
+	getState: () => mockedState_NoC_A,
+	subscribe: jest.fn(),
+	dispatch: jest.fn(),
+};
+
+test('Courses should display amount of CourseCard equal length of courses array', () => {
```

Comment:
usually we start the tests with the word should, no need to mention component because you write tests for specific component. this is relevant for all your tests description

Reply:
Changed